benefit/data/sheets/CaseEvent/CaseEvent.json

```diff
@@ -1412,7 +1423,7 @@
     "ID": "processReasonableAdjustment",
     "Name": "Process reasonable adjustment",
     "Description": "Process reasonable adjustment",
-    "DisplayOrder": 113,
+    "DisplayOrder": 119,
```

Comment:
All display with order 113 should be updated with same value. Same as first comment